mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -175,6 +193,20 @@ export class ExperimentPage extends Component {
         Utils.logErrorAndNotifyUser(e);
         this.setState({ nextPageToken: null, loadingMore: false });
       });
+    this.props
+      .listAllColumnsApi(this.props.experimentId,
+        lifecycleFilterToRunViewType(lifecycleFilterInput))
+      .then((response = {}) => {
+        this.setState({
+          metricsList: response.value.metrics,
+          tagsList: response.value.tags,
+          paramsList: response.value.params,
+        });
+      }
+      )
+      .catch((e) => {
+        Utils.logErrorAndNotifyUser(e);
+      });
```

Comment:
not sure about JS good practises, this code is duplicated, maybe make it a method ?

Reply:
Where is duplicated ? The contents of the blocks differ actually. (actually I followed the same pattern as searchRunsApi

Reply:
same block as lines 69 -> 81, with experimentId <- this.props.experimentId and viewType <- lifecycleFilterToRunViewType(lifecycleFilterInput)

But as I said, I don't know JS enough to know if it's doable to write a function that takes 2 arguments and implement a single block for the 2 cases

Will accept the review as is, we can discuss later when I know more JS